doc/zmq.md

```diff
@@ -111,7 +111,9 @@ using other means such as firewalling.
 
 Note that when the block chain tip changes, a reorganisation may occur
 and just the tip will be notified. It is up to the subscriber to
-retrieve the chain from the last known block to the new tip.
+retrieve the chain from the last known block to the new tip. Also note
+that no notification occurs if the tip was in the active chain - this
+is the case after calling invalidateblock RPC.
```

Comment:
`invalidateblock` is not for production purposes, so I'd rather not mention that here. At most I'd mention this in the invalidateblock rpc itself.

Reply:
Good point, I'll improve `invalidateblock` help then.

Reply:
Done.